rs/tests/nested/BUILD.bazel

```
@@ -88,7 +90,9 @@ system_test_nns(
     uses_guestos_dev = True,
     uses_guestos_dev_test = True,
     uses_setupos_dev = True,
-    runtime_deps = GUESTOS_RUNTIME_DEPS + IC_GATEWAY_RUNTIME_DEPS,
+    runtime_deps = GUESTOS_RUNTIME_DEPS + IC_GATEWAY_RUNTIME_DEPS + [
+        "//rs/tests:vector_with_log_fetcher_image",
+    ],
```

Similarly I would move the runtime dependencies of vector `VECTOR_RUNTIME_DEPS` to `rs/tests:common.bzl` such that we can write:

```suggestion
    runtime_deps = GUESTOS_RUNTIME_DEPS + IC_GATEWAY_RUNTIME_DEPS + VECTOR_RUNTIME_DEPS,
```